Not compatible with Mongoose 4.0 or up.

I got this:
npm WARN peerDependencies The peer dependency mongoose@~4.0.0 included from mongoose-auto-increment will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "mongoose-auto-increment" "--save"
npm ERR! node v0.12.4
npm ERR! npm  v2.10.1
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package mongoose does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer mongoose-auto-increment@4.0.0 wants mongoose@~4.0.0


I've to use the latest mongoose and is there anyway that I can make mongoose-auto-increment work with it? or is there any ETA it will compatible with the latest mongoose?

Still 

```js

npm ERR! peerinvalid The package mongoose does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer mongoose-auto-increment@4.0.0 wants mongoose@~4.0.0
`

is there

I'm getting this error when I run `npm install mongoose-auto-increment`:
`npm ERR! peerinvalid The package mongoose does not satisfy its siblings' peerDependencies requirements!`
`npm ERR! peerinvalid Peer mongoose-auto-increment@4.0.0 wants mongoose@~4.0.0`
`npm ERR! System Darwin 14.0.0`
`npm ERR! command "/usr/local/bin/node" "/usr/local/bin/npm" "install"`
`npm ERR! cwd /Users/lucdetellis/repos/emailhat/dev`
`npm ERR! node -v v0.10.40`
`npm ERR! npm -v 1.4.28`
`npm ERR! code EPEERINVALID`

--- a/package.json
+++ b/package.json
@@ -10,13 +10,13 @@
     "extend": "^2.0.0"
   },
   "peerDependencies": {
-    "mongoose": "~4.0.0"
+    "mongoose": "^4.0.0"
   },
   "devDependencies": {
     "async": "*",
     "chai": "*",
     "mocha": "*",
-    "mongoose": "~4.0.0"
+    "mongoose": "^4.0.0"
   },
   "keywords": [
     "mongoose",

